warp/Network/Wai/Handler/Warp.hs

```diff
@@ -219,10 +223,26 @@ setManager x y = y { settingsManager = Just x }
 -- Enabling this cache results in drastic performance improvement for file
 -- transfers.
 --
--- Default value: since 3.0.13, default value is 0, was previously 10
+-- Default value: 0, was previously 10
```

Comment:
Yup, scratch that, the diff display simply wasn't showing the earlier part of the comment.